src/main/kotlin/edu/ics499/team1/app/entities/UserEntity.kt

```diff
@@ -0,0 +1,19 @@
+package edu.ics499.team1.app.entities
+
+import com.fasterxml.jackson.annotation.JsonManagedReference
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "users")
+data class UserEntity(
+    @Id
+    @GeneratedValue
+    val userId: Int = 0,
```

Comment:
Oh we assign 0 here so when we create the Entity from the Domain object, we don't need to provide a value. It will get overwritten when generated though.